packages/front-end/components/Archetype/AttributeForm.tsx

```diff
@@ -1,4 +1,4 @@
-import React, { useMemo, useState } from "react";
+import React, { useEffect, useMemo, useState } from "react";
```

Comment:
line 146 or so:
We should include the <DatePicker> for attribute values that use a date format (i.e. string formatted as "date")

Reply:
done